xchange-kraken/src/main/java/org/knowm/xchange/kraken/KrakenAdapters.java

```diff
@@ -511,6 +512,7 @@ private static InstrumentMetaData adaptPair(
         .tradingFeeCurrency(
             KrakenUtils.translateKrakenCurrencyCode(krakenPair.getFeeVolumeCurrency()))
         .minimumAmount(minimumAmount)
+        .counterMinimumAmount(counterMinimumAmount)
```

Comment:
minor improvement: you can put `kraken.getCostMin()` and save 1 line of code above :smile:

Reply:
I think PR's owner, I don't even see the button to resolve it